bulk.py

```diff
@@ -86,6 +89,11 @@ def _main():
                 api.post("/changes/%s/submit" % change["id"])
         except Exception as e:
             print("Operation failed: %s" % e, file=sys.stderr)
+    if query_options and options.commands:
```

Comment:
One of the reasons that I refactored handling of the options in 2d356c88b3435d65cbb3f33404f273d18786bf31 was so that it will be easier to force addition of specific options if they are not present.  I.e. in this case we can check if `options.commands` is set, and then enforce that the options contains `CURRENT_REVISION` and `DOWNLOAD_COMMANDS` regardless of whether or not the user has explicitly added them.

Reply:
Thanks. I had something similar in mind but discarded it. I'm now about to push a patch that (I think) should properly apply your good point.

Reply:
Done. I also amended the commit message accordingly. Please let me know if I'm missing any needed review process step here...